index.html

```diff
@@ -1194,16 +1197,104 @@
+            class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+            "padding-top: 0.225em; padding-bottom: 0.298em;">a</span></span><span id="MJXc-Node-108"
+            class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+            "padding-top: 0.372em; padding-bottom: 0.298em;">t</span></span></span></span></span><span id="MJXc-Node-109"
+            class="mjx-mo" style=
+            "padding-left: 0.333em; padding-right: 0.333em;"><span class=
+            "mjx-char MJXc-TeX-main-R" style=
+            "padding-top: 0.077em; padding-bottom: 0.298em;">=</span></span><span id="MJXc-Node-110"
+            class="mjx-mfrac"><span class="mjx-box MJXc-stacked" style=
+            "width: 3.478em; padding: 0px 0.12em;"><span class="mjx-numerator"
+            style="width: 3.478em; top: -1.447em;"><span id="MJXc-Node-111"
+            class="mjx-msub"><span class="mjx-base" style=
+            "margin-right: -0.003em;"><span id="MJXc-Node-112" class=
+            "mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+            "padding-top: 0.446em; padding-bottom: 0.298em; padding-right: 0.003em;">d</span></span></span><span class="mjx-sub"
+            style=
+            "font-size: 70.7%; vertical-align: -0.219em; padding-right: 0.071em;"><span id="MJXc-Node-113"
+            class="mjx-mrow" style=""><span id="MJXc-Node-114" class=
+            "mjx-mn"><span class="mjx-char MJXc-TeX-main-R" style=
+            "padding-top: 0.372em; padding-bottom: 0.372em;">16</span></span><span id="MJXc-Node-115"
+            class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+            "padding-top: 0.446em; padding-bottom: 0.298em;">b</span></span><span id="MJXc-Node-116"
+            class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+            "padding-top: 0.446em; padding-bottom: 0.298em;">i</span></span><span id="MJXc-Node-117"
+            class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style=
+            "padding-top: 0.372em; padding-bottom: 0.298em;">t</span></span></span></span></span></span><span class="mjx-denominator"
+            style="width: 3.478em; bottom: -0.778em;"><span id="MJXc-Node-118"
+            class="mjx-mn"><span class="mjx-char MJXc-TeX-main-R" style=
+            "padding-top: 0.372em; padding-bottom: 0.372em;">65535.0</span></span></span><span style="border-bottom: 1.3px solid; top: -0.296em; width: 3.478em;"
+            class="mjx-line"></span></span><span style=
+            "height: 2.225em; vertical-align: -0.778em;" class=
+            "mjx-vsize"></span></span></span></span></span></span></span>
+          </p>
+          <h3>
+            Read the data from a WebGL texture
+          </h3>
+          <p>
+            Here we list some of the possible approaches.
+          </p>
+          <ul>
+            <li>Synchronous readPixels requires the least amount of code and it
```

Comment:
Synchronous readPixels usage. It is the same call as async, just different setup.